srcpkgs/sayonara-player/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'sayonara-player'
+pkgname=sayonara-player
+version=1.7.0.stable3
+revision=1
+build_style=cmake
+makedepends="pkg-config gst-plugins-bad1-devel qt5-devel qt5-tools-devel
```

Comment:
The fact that `xbps-src` automatically detects a `gst-plugins-base1` dependency suggests that `gst-plugins-bad1-devel` is never actually being used, and should be replaced with `gst-plugins-base1-devel`.